ocean_lib/services/service.py

```diff
@@ -42,6 +43,14 @@ def __init__(
         self.compute_values = compute_values
         self.name = name
         self.description = description
+        self.additional_information = None
+
+        if additional_information:
+            self.additional_information = {}
+            self._reserved_names = ["id", "type", "serviceEndpoint"]
```

Comment:
I think you can have anything you want in additionalInformation. Previously with "additional values" or however it was called in v3, the values were added directly inside the DDO, at root level. Now, if we assign the dictionary inside "additionalInformation", then there is no point in checking reserved names. Basically, we do not care to police whatever the user adds there.